lib/BuildSystem/BuildSystem.cpp

```diff
@@ -1526,7 +1573,7 @@ Rule BuildSystemEngineDelegate::lookupRule(const KeyType& keyData) {
     auto it = getBuildDescription().getCommands().find(key.getCommandName());
     if (it == getBuildDescription().getCommands().end()) {
       // If there is no such command, produce an error task.
-      return Rule{
+      return std::unique_ptr<Rule>(new BuildSystemRule(
```

Comment:
We normally use make_unique, don't we?

Reply:
Yeah. make_unique wasn't happy with these, though.  I think perhaps because of the default argument values?